content/en/tracing/trace_collection/custom_instrumentation/android/otel.md

```diff
@@ -0,0 +1,482 @@
+---
+title: Android and Android TV Custom Instrumentation using OpenTelemetry API
+kind: documentation
+description: 'Instrument your Android and Android TV application with OpenTelemetry API, to send traces to Datadog.'
+aliases:
+- /tracing/trace_collection/otel_instrumentation/android/
+- /tracing/trace_collection/custom_instrumentation/otel_instrumentation/android
+further_reading:
+    - link: 'tracing/glossary/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
+    - link: '/opentelemetry/guide/otel_api_tracing_interoperability'
+      tag: 'Documentation'
+      text: 'Interoperability of OpenTelemetry API and Datadog instrumented traces'
+---
+
+{{% otel-custom-instrumentation-lang %}}
+
+## Requirements and limitations
+
+- You will have to download use the dd-sdk-android-trace and dd-sdk-android-trace-otel dependencies starting 
+with 2.12.0+
```

Comment:
/informational

We must make sure during the release time, the real version is updated.